tests/integration/disambiguation/test_disambiguation_core_db_readers.py

```diff
@@ -34,32 +34,34 @@ def test_get_all_curated_signatures(isolated_app):
     TestRecordMetadata.create_from_file(__name__, '792017.json')
 
     expected = {
-        'affiliations': [
-            {
-                'record': {
-                    '$ref': 'http://labs.inspirehep.net/api/institutions/902725',
-                },
-                'value': 'CERN',
-            },
-        ],
-        'curated_relation': True,
-        'full_name': 'Ellis, John R.',
-        'ids': [
-            {
-                'schema': 'INSPIRE ID',
-                'value': 'INSPIRE-00146525',
-            },
-        ],
-        'raw_affiliations': [
-            {
-                'value': 'CERN. Department of Physics. TH Division Geneva, Switzerland',
-            },
-        ],
-        'record': {
-            '$ref': 'http://labs.inspirehep.net/api/authors/1010819',
+        'author_affiliation': 'CERN',
+        'author_id': 1010819,
+        'author_name': 'Ellis, John R.',
+        'publication': {
+            'abstract': 'This talk describes past progress in probing the structure of matter and the content of the Universe, which has led to the Standard Model of elementary particles, and the prospects for establishing new physics beyond the Standard Model using the LHC particle collider at CERN.',
+            'authors': ['Ellis, John R.'],
+            'collaborations': [],
+            'keywords': [
+                '29.20.Db',
+                '98.80.-k',
+                '12.60.-i',
+                'elementary particles',
+                'standard model',
+                'cosmology',
+                'particle accelerators',
+                'Elementary particles',
+                'Beyond the Standard Model',
+                'Colliders',
+                'Large Hadron Collider',
+                'Standard Model',
+            ],
+            'publication_id': 792017,
+            'title': 'The quest for elementary particles',
+            'topics': ['Phenomenology-HEP'],
         },
+        'publication_id': 792017,
         'signature_block': 'ELj',
-        'uuid': '94f560d2-6791-43ec-a379-d3dc4ad0ceb7',
+        'signature_id': '94f560d2-6791-43ec-a379-d3dc4ad0ceb7',
```

Comment:
I'm really tempted to deviate from the paper here and call it a `signature_uuid`...

Reply:
That sounds reasonable. Will require a few changes on my side, but this would be more consistent with INSPIRE naming conventions.